subsys/bluetooth/host/gatt.c

```diff
@@ -5210,6 +5210,71 @@ int bt_gatt_store_ccc(uint8_t id, const bt_addr_le_t *addr)
 	return 0;
 }
 
+/** Struct used to store both the id and the random address of a device when replacing
+ * random addresses in the ccc attribute's cfg array with the device's id address after
+ * pairing complete.
+ */
+struct convert_to_id_on_match_params {
```

Comment:
Please come up with some shorter name for this

Reply:
E.g. simply `struct addr_match` would have been fine (feel free to suggest something else if you don't like it - as long as it's short, yet understandable)

Reply:
Okidoki.

Reply:
Changing it to addr_match.

Reply:
Used addr_match.